afl-fuzz.c

```diff
@@ -7071,6 +7074,7 @@ static void check_term_size(void) {
 
   if (ioctl(1, TIOCGWINSZ, &ws)) return;
 
+  if (ws.ws_row == 0 && ws.ws_col == 0) return;
```

Comment:
What's the purpose of this change?

Reply:
In some version of tty like putty, when I run afl-fuzz, the status screen won't show no matter how large the window is , after check, I found that ws.ws_row and ws_ws_col always returns 0.

Reply:
ACK